src/main/java/org/employable/Controller/RecruiterController.java

```diff
@@ -30,7 +30,7 @@ public void setContactInfo(String email){
   }
 
   public void createJobListing(String position, String company, String link, String location){
-    JobListingModel model = new JobListingModel(position, company, link, location);
+    model.addJobListing(position, company, link, location);
```

Comment:
Thanks for this! I like calling this method in the model rather than instantiating job listing right here! :-)